actions/lib/base.py

```diff
@@ -11,17 +11,27 @@ def __init__(self, config):
         self._init_dd()
 
     def _init_dd(self):
-        options = {
-            'api_key': self.config['api_key'],
-            'app_key': self.config['app_key']
-        }
+        options = {}
+
+        if 'api_key' in self.config and 'app_key' in self.config:
+                options = {
+                    'api_key': self.config['api_key'],
+                    'app_key': self.config['app_key']
+                }
         initialize(**options)
 
     @abstractmethod
     def _run(self, **kwargs):
         pass
 
     def run(self, **kwargs):
+        if 'api_key' in kwargs and 'app_key' in kwargs and kwargs['api_key'] and kwargs['app_key']:
```

Comment:
Why not using kwargs.get('api_key') and kwargs.get('app_key') ?

Reply:
Because I'm not a dev, so my code is bad...
Thanx for the advice. I'll re-do the pull when corrected the stuff.  ^_^